src/v/pandaproxy/schema_registry/types.cc

```diff
@@ -59,7 +62,8 @@ std::ostream& operator<<(
       os,
       "type: {}, definition: {}, references: {}",
       to_string_view(def.type()),
-      def.raw(),
+      // TODO BP: Prevent this linearization
```

Comment:
q: here and elsewhere, what do you mean by linearization?

Reply:
> q: here and elsewhere, what do you mean by linearization?

The operation that allows a signature of `(pointer, len)`, and requires contiguous storage, resulting in oversize-allocation.